msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -632,6 +632,8 @@ pvt_select_msg_recipient_replyall = You must select the reply all recipient befo
 
 pvt_missing_subject = You must enter a subject before you may send this message.
 pvt_missing_subject_draft =You must enter a subject before you may save this message.
+pvt_missing_body = You must enter a body before you may send this message.
```

Comment:
I'd prefer to change the wording of "enter a body" to something else to make it clearer what is being asked here. Something like "You must write your message before you can send it."